feeds/wlan-ap/wlan-ap-config/files/etc/hotplug.d/iface/40-fix-lan-wan-ip-conflict

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 [ "$ACTION" = ifup -o "$ACTION" = ifupdate ] || exit 0
-[ "$INTERFACE" = wan ] || exit 0
```

Comment:
How about just executing the script for interface lan along with wan? Does this help? With the above change, the script will also be unnecessary executed for wlan interfaces?

Reply:
So you are suggesting: [ "$INTERFACE" = wan ] || [ "$INTERFACE" = lan ] || exit 0

This avoids the ubus calls and parsing of results.  Beyond that wlan interface changes will not find any new conflict and should take no action.  Does keep behaviour closer to original and still addresses issue.

Reply:
Sounds good, that's what I'll do 👍